Gemfile

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-source 'https://rubygems.org'
+source 'http://rubygems.org'
```

Comment:
We shouldn't need to switch to http. Is this to fix the ssl error that we see? Check for other workarounds. Try once again with https.